hyperspy/docstrings/plot.py

```diff
@@ -49,7 +49,7 @@
         `navigator_shape` of the current object (for a dynamic navigator).
         If the signal `dtype` is RGB or RGBA this parameter has no effect and 
         the value is always set to ``'slider'``.
-    axes_manager : None or ~hyperspy.axes.AxesManager
+    axes_manager : None or :py:class:`~hyperspy.axes.AxesManager`
```

Comment:
I guess this is a matter of style. Do we have a preference or precedence for which to choose? If you don't use the `:py:class:` prefix, sphinx is smart enough to still link it, but it links as a regular hyperlink, instead of monospaced:

![image](https://user-images.githubusercontent.com/1278301/60518451-8e2c7980-9caf-11e9-93b4-9bf4625d2d00.png)

Reply:
This is to get a better display in the spyder help panel:

without the `:py:class:`:

![image](https://user-images.githubusercontent.com/11851990/60519408-6521de00-9cdb-11e9-8807-0fdfb3e938b6.png)


with the `:py:class:`

![image](https://user-images.githubusercontent.com/11851990/60518955-74545c00-9cda-11e9-9287-285fb270c223.png)

Unfortunately, this doesn't change much in the notebook... but maybe at some point, there will be better html rendering of the docstring. Maybe this is already a plugin to do that, I would have missed?

Reply:
Oh interesting. I hadn't considered what it actually looks like in the programming environments, only the HTML output :flushed:. 

If you want, I can go through and change the others that I styled this way?

Reply:
I think I already did, if I haven't missed any! I started to look at tweaking other docstring, but I am afraid that would be a big rabbit hole...

Reply:
One thing I did notice in the HTML output is if you use a fully resolved sphinx link as a parameter or return type, the other "basic" types do not get linked:

e.g. I made `BaseSignal` a full sphinx link, and the `str` and `None` types did not get linked:

![image](https://user-images.githubusercontent.com/1278301/60526355-36e1d580-9cbe-11e9-8ef4-82b4f14e2174.png)